test/java/seedu/addressbook/storage/InvalidData.txt

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AddressBook>
+    <persons>
+        <invalid>data</invalid>
+    </persons>
+</AddressBook>
```

Comment:
Test data should be in a different folder, not in the same folder as java files. See higher level addressbook projects for reference.